### DISCUSSION TITLE
Some question about output file

### DISCUSSION THREAD

**Discussion by Yichun886** [hernanchavezthielemann/GRO2LAM#47] [Q&A] [answered]
**Describe the bug**
In my top file, I have 100 molecules (1200 atoms), but there is only one molecule (12 atoms) in the data file. Do I use the software wrongly? Thanks.

**Screenshots**
![543da63d0dd24153a57de6ab229673d](https://user-images.githubusercontent.com/46417144/88377529-f45fcf00-cdd1-11ea-8e4d-9369ed7941c1.png)
![b23ff25d78655e458a36089d0a7a45c](https://user-images.githubusercontent.com/46417144/88377611-11949d80-cdd2-11ea-912f-e4b682568d87.png)
![4358fa5f2b5f944a14a32ac4323bbb3](https://user-images.githubusercontent.com/46417144/88377631-16f1e800-cdd2-11ea-80cc-ed3ef5f61e78.png)

**Comment by hernanchavezthielemann** [comment]
Hi Yi @Yichun886,

With this small amount of information I could just say that your gro file is missing the molecule identifier. But I'm not sure whether this is causing the issue or if it is something else.
Could you post all your files?

Best,
Hernan

**Comment by Yichun886** [comment]
Hi Hernan @hernanchavezthielemann ,

Thank you for your reply very much.

There are 1000 identical molecules in one box. The following is my files.

Best wishes

Yichun
[files.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/4971526/files.zip)

**Comment by hernanchavezthielemann** [comment]
Hi Yichun,

There were some things, please take a look on the attachment.
As I previously mentioned, I have to add the residue tag in .gro file, Also I split your top file.


[Yichun.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/4981315/Yichun.zip)

Best,
Hernan

**Comment by Yichun886** [comment]
Hi Hernan @hernanchavezthielemann ,

The problem is solved.
Thank you very much.

Best wishes
Yichun

**Comment by Yichun886** [comment]
Hi Hernan @hernanchavezthielemann ,

Sorry for disturbing you again.

I have a problem to convert the gro file to data file. There are no atoms, bonds, and angles in the data file.
Would you please help me to look this.

Thanks
Yichun
[c9h20.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/5027288/c9h20.zip)

**Comment by hernanchavezthielemann** [comment]
Hi @Yichun886, 
If you split the top like before - look at the attachment -  it works. If you cannot make it work let me know.

I will try to look at it more in detail tomorrow.

Best!
Hernan

[yi2.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/5031417/yi2.zip)

**Comment by Yichun886** [comment]
Hi Hernan @hernanchavezthielemann 

Thank you for yout reply.

I have tested your file for many times, and can't get the same results with yours.
Then, I try to calculate the files in the example file, however, the result is the same as what I had in the first time, i.e., the number of atoms in the output  don't match the gro file. (Please see the attachment)

If I split the top file, I'll get 0 atoms in the output file.

I think there is something wrong in my code or system. I try ubuntu 18, ubuntu 16 and centos 7. I get the same results. I use the code from the 27ene19 branch. Would please tell me which system you use, and can I get your codes?

Best wishes
Yichun
[data.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/5041012/data.zip)

**Comment by hernanchavezthielemann** [comment, marked as answer]
Hi @Yichun886,

Don't worry, maybe you are not pressing autoload button? there should be a warning at the beginning If you go without it.
If I try without it I obtain the same as you I think.

This is what I am doing (splitting!):
[yi2_c9h20.zip](https://github.com/hernanchavezthielemann/GRO2LAM/files/5042671/yi2_c9h20.zip)
![Yichun](https://user-images.githubusercontent.com/29739618/89668858-c2c12a80-d8de-11ea-9b3d-027f6a03a620.gif)

Ubuntu 18 or 16 is fine, and Python 2.7.

If you are obtaining something out then it is not a machine problem.

Let me know how it goes.

Best,
Hernan

**Comment by Yichun886** [comment]
Hi Hernan @hernanchavezthielemann ，

You're right I load the files manually, and I get this problem. After I press the autoload button, the problem is solved.

Thank you very much for helping me.

Best wishes
Yichun

**Comment by hernanchavezthielemann** [comment]
Great @Yichun886!, you are welcome 
And if you think that it is not clear how to use it or if it is confusing, let me know what you think, or even if you have any other feedback, so I can improve it.
Best regards,
Hernan
